Makefile

```diff
@@ -35,3 +35,15 @@ $(call add-bindata,etcd,./bindata/etcd/...,bindata,etcd_assets,pkg/operator/etcd
 # See vendor/github.com/openshift/build-machinery-go/scripts/run-telepresence.sh for usage and configuration details
 export TP_DEPLOYMENT_YAML ?=./manifests/0000_12_etcd-operator_06_deployment.yaml
 export TP_CMD_PATH ?=./cmd/cluster-etcd-operator
+
+# This was copied from https://github.com/openshift/machine-config-operator
+# Run tests
+test: test-unit test-e2e
+
+# Unit tests only (no active cluster required)
+test-unit:
+	CGO_ENABLED=0 go test -tags=$(GOTAGS) -count=1 -v ./cmd/... ./pkg/... 
+
+# This was copied from https://github.com/openshift/cluster-image-registry-operator
+test-e2e:
+	go test -failfast -timeout 120m -v$${WHAT:+ -run="$$WHAT"} ./test/e2e/
```

Comment:
is `WHAT` a think here?  If so, did you want to land docs similar to what origin [used to have][1]?  Or somewhere explain to folks how it is expected to be used?

[1]: https://github.com/openshift/origin/commit/82d3e63a7e267292a28547b51bc0b79f2c7e2213#diff-b67911656ef5d18c4ae36cb6741b7965L147

Reply:
I can removed. I think `go test` allows a regex pattern to be specified to only run a subset of tests. I can remove it, if you like.

```
-run regexp
        Run only those tests and examples matching the regular expression.
        For tests, the regular expression is split by unbracketed slash (/)
        characters into a sequence of regular expressions, and each part
        of a test's identifier must match the corresponding element in
        the sequence, if any. Note that possible parents of matches are
        run too, so that -run=X/Y matches and runs and reports the result
        of all tests matching X, even those without sub-tests matching Y,
        because it must run them to look for those sub-tests.
```

Reply:
I deleted the overriding rules for test-e2e and test-unit and replaced them with definitions similar to what the Makefile from https://github.com/openshift/cluster-kube-apiserver-operator has. Let's wait for the tests to succeed again.